src/utils/docs.jl

```diff
@@ -19,6 +19,14 @@ function doctestsetup()
   return :(using Oscar; Oscar.AbstractAlgebra.set_current_module(@__MODULE__))
 end
 
+function doctestfilters()
+  # this returns a list of doctest filters that should be passed to all doctest invocations
+  return [
+    r"0-element SubObjectIterator{RayVector{QQFieldElem}}|RayVector{QQFieldElem}\[\]" => "RayVector{QQFieldElem}[]", # due to JuliaLang/julia#57898, oscar-system/Oscar.jl#4872
+    r"0-element SubObjectIterator{PointVector{QQFieldElem}}|PointVector{QQFieldElem}\[\]" => "PointVector{QQFieldElem}[]", # due to JuliaLang/julia#57898, oscar-system/Oscar.jl#4872
```

Comment:
```suggestion
    # due to JuliaLang/julia#57898, oscar-system/Oscar.jl#4872:
    r"^0-element SubObjectIterator{RayVector{QQFieldElem}}|^RayVector{QQFieldElem}\[\]"m => "RayVector{QQFieldElem}[]",
    r"^0-element SubObjectIterator{PointVector{QQFieldElem}}|^PointVector{QQFieldElem}\[\]"m => "PointVector{QQFieldElem}[]",
    r"^ 0-element SubObjectIterator{RayVector{QQFieldElem}}|^ \[\]"m => " []",
    r"^ 0-element SubObjectIterator{PointVector{QQFieldElem}}|^ \[\]"m => " []",
```
It's not very nice but this might work.